libr/core/cmd_seek.c

```diff
@@ -341,6 +341,15 @@ static int cmd_seek(void *data, const char *input) {
 				off = core->lines_cache[curr+sl_arg];
 				r_core_seek (core, off, 1);
 				break;
+			case 'c':
+				R_FREE (core->lines_cache);
```

Comment:
What about cachesz=-1 too? Put this in a separe function. Maybe move all the line stuff into core/line.c?